Vizzuality/heco-invest
# Improve the appearance of Storybook

This PR makes two changes to Storybook:

1. The preview now has a light background colour which is the same used on the platform
2. The description and types of the props are shown in the “Controls” panel by default

![The preview has a light background and the descriptions and types of the props are shown in the controls panel](https://user-images.githubusercontent.com/6073968/158765841-66c54e7a-41b2-42ea-ba06-0f520846f4b6.png)

## Testing instructions

Make sure:
- the background of the preview window uses the “background-light” colour
- the rest of the UI does not use this colour as background
- the description and types of the props are shown by default in the “Controls” panel

## Tracking

Not tracked.
